packages/components/header/template.njk

```diff
@@ -81,6 +81,10 @@
           <a class="nhsuk-header__account-link" href="{{ item.href }}">
             {{ _accountItem(item) }}
           </a>
+        {% elif item.action %}
+          <button class="nhsuk-header__account-button" formaction="{{ item.action }}" formmethod="{{ item.method or "post" }}">
```

Comment:
Should this have ‘role=link‘ so that it behaves more like how it looks (eg will respond to “click link log out” for voice control)? 🤔

Reply:
I quickly peeked at the spec, and this role seems aimed at those using non-semantic elements for hyperlinks; a button is a button, no?

Reply:
I'm not sure. The NHS button adds `role="button"` to links that are styled as buttons, and the GOVUK button additionally [adds some javascript](https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/button/button.mjs#L29-L57) so that links-styled-as-buttons get clicked if the spacebar is hit when they’re focused.

Neither the GOVUK nor NHS frontend current supports buttons-styled-like-links at the moment, but there’s a backlog discussion about it here: https://github.com/alphagov/govuk-design-system-backlog/issues/159

In our case, maybe it’s a bit less relevant as the "Log out" item in the header kinda looks half-way between a link and button (especially if the underlines get removed in #1136?